.depcheckrc

```diff
@@ -23,3 +23,10 @@ ignores:
  - style-loader
  - ts-loader
  - mongodb-runner
+ - "@wdio/local-runner"
+ - "@wdio/mocha-framework"
+ - "@wdio/spec-reporter"
+ - wdio-vscode-service
+ - wdio-wait-for
+ - "@wdio/globals"
+ - "@wdio/types"
```

Comment:
In Compass and mongosh our depcheckrcs often have comments about _why_ a specific dependency is in the ignore list, this might be an opportunity to also introduce this pattern here 🙂

Reply:
Makes sense - I will make sure to do that.